pkg/api-server/server.go

```diff
@@ -96,6 +103,9 @@ func (a *ApiServer) Start(stop <-chan struct{}) error {
 }
 
 func SetupServer(rt runtime.Runtime) error {
-	apiServer := NewApiServer(rt.ResourceManager(), definitions.All, rt.Config().ApiServer)
+	apiServer, err := NewApiServer(rt.ResourceManager(), definitions.All, rt.Config())
```

Comment:
I would prefer to avoid dependency of `api-server` on `cp-config`.

For `/config` endpoint, it's not important whether `config` parameter is a struct or interface.